app/Components/Game/index.js

```diff
@@ -77,9 +77,10 @@ class Game extends React.Component {
   }
 
   render() {
+    var studentID = 0;
```

Comment:
No -- it's just hard coded for now. Later on, I'll implement randomisation so that a different student profile will be picked at random (along with different set of questions).